R/run.R

```diff
@@ -806,7 +813,9 @@ CmdStanMCMCProcs <- R6::R6Class(
           }
           if (grepl("Gradient evaluation took",line, fixed = TRUE)
               || grepl("leapfrog steps per transition would take",line, fixed = TRUE)
-              || grepl("Adjust your expectations accordingly!",line, fixed = TRUE)) {
+              || grepl("Adjust your expectations accordingly!",line, fixed = TRUE)
+              || grepl("stanc_version",line, fixed = TRUE)
+              || grepl("stancflags",line, fixed = TRUE)) {
```

Comment:
This is a bit of a workaround. Will make a more sustainable fix for the actual release.